packages/grafana-data/src/transformations/transformers/renameByRegex.test.ts

```diff
@@ -147,7 +200,6 @@ describe('Rename By Regex Transformer', () => {
           Array [
             Object {
               "config": Object {
-                "displayName": "Time",
```

Comment:
Why was this removed? 🤔

Reply:
ah ok so it's to do with the subtle differences in the different regexp implementations.

with the old way, `regex.test(displayName)` is truthy so it goes on to do the replace logic (which ultimately does nothing because it's replacing an empty string...): https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/transformations/transformers/renameByRegex.ts#L53

with the new logic, `regex.test(displayName)` is falsey so the code immediately short-circuits and returns the field as it was.